kinto/core/permission/memory.py

```diff
@@ -58,7 +58,8 @@ def remove_principal(self, principal):
     def get_user_principals(self, user_id):
         user_key = 'user:%s' % user_id
         members = self._store.get(user_key, set())
-        return members
+        group_authenticated = self._store.get('user:system.Authenticated', set())
```

Comment:
Oh yes I do actually, we are trying to get the groups linked to system.Authenticated.